SETUP/DEVELOPMENT.md

```diff
@@ -0,0 +1,79 @@
+# Code Development
+
+This doc is designed for users who want to develop on the DProofreaders (DP)
+codebase.
+
+## Introduction
+
+The DP code is almost entirely server-side PHP with a smattering of JS and
+CSS. It uses a MySQL backend database and requires a web server -- pgdp.net
+uses Apache but Nginx should work as well. See the [installation docs](INSTALL.md)
+for information on minimum middleware versions.
+
+The code does not use a PHP framework or an ORM. It was developed 20 years ago
+and has grown very organically over the years. There has been effort in the
+past decade to standardize things but you'll find many different patterns remain.
```

Comment:
I wonder if it would be a good idea to link to [the code style page](https://github.com/DistributedProofreaders/dproofreaders/blob/master/SETUP/CODE_STYLE.md) or [best practice](https://www.pgdp.net/wiki/DP_Code_Best_Practices) that outlines some of those patterns.

A a side note, if there are more expected patterns, it may be worth expanding on the list in a follow-up PR (coming to mind would be getting GET/POST parameters, avoiding boolean in DB and favoring `tinyint` enums (maybe?)). As a newcomer, those are not necessarily obvious due to a mix of new and older deprecated patterns mixing.